docker-compose/camel-toolbox-config/configuration.properties

```diff
@@ -1,12 +1,13 @@
-fcrepo.baseUrl=http://fcrepo:8080/fcrepo/rest
-fcrepo.authHost=fcrepo
+fcrepo.baseUrl=http://localhost:8080/fcrepo/rest
+fcrepo.authHost=localhost
 
-jms.brokerUrl=tcp://fcrepo:61616
+jms.brokerUrl=tcp://localhost:61616
 
 solr.indexing.enabled=true
-solr.baseUrl=http://solr:8983/solr/fcrepo
+solr.baseUrl=http://localhost:8983/solr/fcrepo
+solr.fcrepo.defaultTransform=org/fcrepo/camel/indexing/solr/default_transform.xsl
 
 triplestore.indexing.enabled=true
-triplestore.baseUrl=http://fuseki:3030/fcrepo
+triplestore.baseUrl=http://localhost:3030/fcrepo
```

Comment:
These refer to the individual containers in the `docker-compose.yml` file, by switching them to localhost it no longer works as a single `docker compose up`. I had to switch them back to test it.